### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kevinyamauchi**
this PR updates the message on the readme to indicate that we are currently under review and that changes are to be expected as we go through that process.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Work in progress ⚠
+# In review ⚠
 
--   **The library is not ready.** We aim at a beta release in the following months. If interested in a demo/early beta, please reach out to us.
+-   **The library is currently under review.** We expect there to be changes as the community provides feedback. If interested in a demo/early beta, please reach out to us.
```
Comment (LucaMarconato) [inline README.md:3]: I would remove "If interested in a demo/early beta, please reach out to us.", otherwise good to go!

**Review by LucaMarconato — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (giovp) [reply README.md:3]: same!

**Comment by giovp** [comment]
so in the landing page of docs there is no reference to being a beta
image

**Comment by kevinyamauchi** [comment]
Screenshot 2023-05-05 at 09 02 47

I added this note to the docs

**Comment by kevinyamauchi** [comment]
image2

**Comment by kevinyamauchi** [comment]
I also updated the installation instructions. Since @LucaMarconato approved and I addressed @giovp 's comments, I will merge once test pass.

**Pull request merged**